CSC322-Grinnell/mayflower-directory
# Externalize gallery

This PR does not address any open issues. 

# Changes

- isolates gallery items into a shared/separate view using ruby `partials`, which works and renders the way the header and footer do. 

# Testing

1. `git checkout externalize-gallery`
2. `rails s -b $IP -p $PORT`
2.  Go to the *Resource* tab
3. Make sure the links work
3. Make sure the formatting looks consistent and not broken

# Note

This PR is to be used for departments-html to generate individual department gallery "cards" under departments tab.
